### PR TITLE
remove all reviews associated with a product when product is deleted

### LINKED ISSUE #13: Remove All Reviews associated with a product when the product is deleted

### PR DESCRIPTION
**Pull request by khareemah**
Please review

### CODE REVIEW

File: controllers/reviewController.js [hunk 1]
```diff
@@ -74,9 +74,9 @@ const deleteReview = async (req, res) => {
 const getSingleProductReviews = async (req, res) => {
```
Comment (fisayo-dmg) [inline controllers/reviewController.js]: what happens when the product id does not exist?

**Review by fisayo-dmg — CHANGES_REQUESTED** [review]

File: controllers/reviewController.js (hunk 1 above)
Comment (khareemah) [reply controllers/reviewController.js]: An empty array would be returned

Comment (fisayo-dmg) [reply controllers/reviewController.js]: but in many other parts of the code you have returned an error with a message that the product doesn't exist, any reason why you are not keeping it consistent with that?

Comment (khareemah) [reply controllers/reviewController.js]: That's true, I'll uncomment the if statement

**Review by fisayo-dmg — APPROVED** [review]

**Pull request merged**